script/upgrades/dependencies.json

```diff
@@ -35,6 +37,7 @@
   },
   "44787": {
     "BridgedUSDC": "0x87D61dA3d668797786D73BC674F053f87111570d",
+    "NativeUSDC": "0x17dAe16f0EcB55FCd6887b01C59aD1F5e592d490",
```

Comment:
Pending to see if we can replace this by the actual USDC deployed on Alfajores https://developers.circle.com/stablecoins/docs/usdc-on-test-networks